### DISCUSSION TITLE
Is there a web3-unleashed category?

### DISCUSSION THREAD

**Discussion by insanity54** [ConsenSys-archive/truffle#5716] [General Q&A] [answered]
[This guide](https://trufflesuite.com/guides/nft-royalty/) on trufflesuite.com links to https://github.com/orgs/trufflesuite/discussions/categories/web3-unleashed which leads me to believe that there is a web3-unleashed category. I'm not seeing one. Is this intentional? Is the web3-unleashed category missing? I would love a web3-unleashed category to post under.

**Comment by apdan510** [comment]
@emilyJLin95 this [link ](https://github.com/orgs/trufflesuite/discussions/categories/web3-unleashed)is not going to the the Web3 Unleashed discussions category. Any thoughts on this one?

Thanks!

**Comment by emilyJLin95** [comment, marked as answer]
Whoops sorry about that - i think I might've accidentally deleted it at some point
https://github.com/orgs/trufflesuite/discussions/categories/web3-unleashed

Here ya go!
